test/smoke/packages/grafana/7.5.7/grafana-test.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Checking package is installed or not
+
+tanzu package installed list | grep "local-path-storage.community.tanzu.vmware.com" || {
+  version=$(tanzu package available list local-path-storage.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+  tanzu package install local-path-storage --package-name local-path-storage.community.tanzu.vmware.com --version "${version}"
+}
+
+tanzu package installed list | grep "contour.community.tanzu.vmware.com" || {
+    version=$(tanzu package available list contour.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+    tanzu package install contour --package-name contour.community.tanzu.vmware.com -f "${MY_DIR}"/../../contour/1.18.2/contour-values.yaml --version "${version}"
```

Comment:
version is 1.18.2 in the path, so `${version}` is not needed here ?

also, refer the path from repo root dir instead of `${MY_DIR}`

Reply:
Right `${version}` is an appropriate choice. Just want a suggestion can we skip putting the smoke test in `smoke/packages/package-name/` instead of `smoke/packages/package-name/version` as these smoke-tests are testing for the latest stable version of the package by this we do not have to insert version dir and add smoke test whenever there is a new version. What’s your opinion?

Reply:
lets keep it versioned